Code/GraphMol/Descriptors/MolSurf.cpp

```diff
@@ -357,5 +361,23 @@ std::vector<double> calcPEOE_VSA(const ROMol &mol, std::vector<double> *bins,
 
   return res;
 }
+
+std::vector<double> calcCustomProp_VSA(const ROMol &mol, const std::string &customPropName,
+		                         std::vector<double> *bins, bool force) {
```

Comment:
since bins is never checked for null, I would make it 
```
const std::vector<double> &bins, 
```

Reply:
Given that `bins` isn't optional, I would go with Brian's suggestion and just make it a `const std::vector<double> &`